gauntlet/packages/gauntlet-solana-contracts/src/commands/abstract/close.ts

```diff
@@ -0,0 +1,67 @@
+import { logger, prompt } from '@chainlink/gauntlet-core/dist/utils'
+import { SolanaCommand } from '@chainlink/gauntlet-solana'
+import { PublicKey, TransactionInstruction } from '@solana/web3.js'
+import { CONTRACT_LIST, getContract } from '../../lib/contracts'
+
+export default abstract class Close extends SolanaCommand {
```

Comment:
@krebernisak I went with this way of making a generic Close command, instead of wrapping the class into a closure.